src/repl_interface/repl_interface.cr

```diff
@@ -12,7 +12,6 @@ module IC::ReplInterface
     @auto_completion = AutoCompletionHandler.new
     @history = History.new
     @prelude_complete_channel : Channel(Int32)? = nil
-    @prelude_complete = true
```

Comment:
Turns out to not be needed, can check if the channel 1) exists and 2) hasn't been closed yet instead.